Makefile

```diff
@@ -4,22 +4,40 @@ default: build
 build:
 	docker build -t rpi-homebridge-dev .
 
+clean:
+	-docker kill homebridge
+	-docker rm homebridge
+
 # Target to build and run and subsequently remove image
 run: build
 	docker run --net=host --rm \
-		-p "localhost:51826:51826"
+		-p "51826:51826" \
 		-v "$(shell pwd)/config.json:/root/.homebridge/config.json" \
 		-v "$(shell pwd)/plugins.txt:/root/.homebridge/plugins.txt" \
+		-v "$(shell pwd)/persist:/root/.homebridge/persist" \
 		rpi-homebridge-dev
 
 # Target to drop into an interractive shell
 shell: build
 	docker run --net=host --rm \
-		-p "localhost:51826:51826"
+		-p "51826:51826" \
 		-v "$(shell pwd)/config.json:/root/.homebridge/config.json" \
 		-v "$(shell pwd)/plugins.txt:/root/.homebridge/plugins.txt" \
+		-v "$(shell pwd)/persist:/root/.homebridge/persist" \
 		-it rpi-homebridge-dev bash
 
+# Target to buld and run in detached mode (continuously)
+go: build
+	make clean
```

Comment:
This shouldn't need to be here. If you think it needs to clean, it could go as a dependency before build.

Additionally, I only use the `Makefile` myself for testing. I use a `docker-compose.yml` file for actually running the service since it takes care of container management for me.

Reply:
i tend to launch with `make go` each time i redeploy.  Especially if i need to rebuild home bridge when something randomly decides to break.  A `make clean` just ensures that I have a clean build being deployed and that every dependency  gets updated.  Although it does make those redeploys slower as a result.

Reply:
It just seems off to have `clean` a dependency of `go`, when it's not strictly so.

You could always use `make clean go` to deploy. Alternately, `docker-compose up -d` since that will recreate the image and/or container if necessary. That said, you should add the persist volume to the compose file too.

Reply:
huh, i wasn't aware i could use `make clean go`.  I will give that an attempt and if it works i'll update my PR.  Thanks for the feedback.

I'll also update the docker-compose with the persists volume.  Sorry missed it since i wasn't using the compose.